README.md

```diff
@@ -0,0 +1,214 @@
+
```

Comment:
- [ ] Kindly, include in your instructions the name of your test user "John", so your users can test your app without having to create users in the Rails console. 👍